index.html

```diff
@@ -181,134 +189,140 @@ <h1>
+        </div>
+        <ul class="my-skills">
+          <li class="language-skills">
+            <div class="language-title">
+              Languages
+              <img src="./images/icons/down-arrow.svg" alt="down-arrow" />
             </div>
-            <div class="language-name">
-              <img src="./images/language-icons/css.svg" alt="" />
-              CSS
+            <div class="language-names">
+              <div class="language-name">
+                <img src="./images/language-icons/javascript.svg" alt="" />
+                Javascript
+              </div>
+              <div class="language-name">
+                <img src="./images/language-icons/html.svg" alt="" />
+                HTML
+              </div>
+              <div class="language-name">
+                <img src="./images/language-icons/css.svg" alt="" />
+                Css
+              </div>
             </div>
-          </div>
-        </li>
-        <li>
-          Frameworks
-          <img src="./images/icons/right-arrow.svg" alt="right-arrow" />
-        </li>
-        <li>
-          Skills <img src="./images/icons/right-arrow.svg" alt="right-arrow" />
-        </li>
-      </ul>
-    </section>
+          </li>
+          <li>
+            Frameworks
+            <img src="./images/icons/right-arrow.svg" alt="right-arrow" />
+          </li>
+          <li>
+            Skills
+            <img src="./images/icons/right-arrow.svg" alt="right-arrow" />
+          </li>
+        </ul>
+      </section>
 
-    <!-- About Section Ends -->
+      <!-- About Section Ends -->
 
-    <!-- Contact Me Form Starts -->
+      <!-- Contact Me Form Starts -->
 
-    <section class="contact-me">
-      <h2 class="contact-me-tittle">Contact Me</h2>
-      <p class="contact-me-text">
-        If you have an application you are interested in developing, a feature
-        that you need built or a project that needs coding. I’d love to help
-        with it
-      </p>
-      <form action="https://formspree.io/f/xpzerqbg" method="post">
-        <input
-          id="username"
-          type="text"
-          name="username"
-          required
-          maxlength="30"
-          title="Name should be text and not more than 30 characters"
-          placeholder="Enter your Name"
-        />
+      <section class="contact-me">
+        <h2 class="contact-me-tittle">Contact Me</h2>
+        <p class="contact-me-text">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <form action="https://formspree.io/f/xpzerqbg" method="post">
+          <input
+            id="username"
+            type="text"
+            name="username"
+            required
+            maxlength="30"
+            title="Name should be text and not more than 30 characters"
+            placeholder="Enter your Name"
+          />
```

Comment:
- [ ] Excellent work you did on the project 👏 . However, there are a few adjustments you need to make to the background images on this contact area. For more clarity, I did make a comparison below between the Figma design and yours, to see the disparity. Sticking to the design given is essential, as potential clients will ask for the same when you begin to work professionally. Please make the highlighted addion 👍 

## Figma omission points:
![image](https://user-images.githubusercontent.com/50990925/222878339-8c56c145-3816-4376-920c-d2e64680b04d.png)

## your design:
![image](https://user-images.githubusercontent.com/50990925/222878381-80a08ef9-25c6-4a2d-a79e-d0afb9310e2d.png)